at_onboarding_flutter/example/lib/services/at_service.dart

```diff
@@ -62,7 +61,7 @@ class AtService {
 
   Future<bool> makeAtsignPrimary(String atsign) async {
     atsign = formatAtSign(atsign)!;
-    return atClientServiceMap[atsign]!.makeAtSignPrimary(atsign);
+    return _keyChainManager.makeAtSignPrimary(atsign);
```

Comment:
Need to use dependencies override to use this(at_backupkeys_flutter, at_lookup and client sdk and mobile too)